recipes/cell-types-analysis/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.0.7" %}
+
+package: 
+  name: cell-types-analysis
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/ebi-gene-expression-group/cell-types-analysis/archive/{{ version }}.tar.gz
+  sha256: ad16eef0a6736c7460624bb380b3044e4e1c9c2acac8f248181c1f8b5ea6cc80
+
+build:
+  number: 1
+  noarch: generic
+
+requirements:
+    run:
```

Comment:
add r-stringi as suggested? Will need the build bumped for upload to anacoda obvs